Plots/Panels/NCL_panel_10.py

```diff
@@ -0,0 +1,258 @@
+gvutil.set_axes_limits_and_ticks(ax1,
+                                 xlim=[100, 220],
+                                 ylim=[0, 1.55 * 1e16],
+                                 xticks=[135, 180],
+                                 yticks=np.linspace(0, 1.55 * 1e16, 7),
+                                 xticklabels=['135E', '180'],
+                                 yticklabels=np.arange(0, 181, 30))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax1,
+                             x_minor_per_major=3,
+                             y_minor_per_major=3,
+                             labelsize=12)
+
+# Remove tick marks on right side of ax1
+ax1.tick_params('y', which='both', right=False)
+
+# Use geocat.viz.util convenience function to add titles
+gvutil.set_titles_and_labels(ax1,
+                             maintitle="Pacific Region",
+                             lefttitle="Velocity Potential",
+                             righttitle="m2/s",
+                             ylabel="elapsed time")
+
+# Format axes for zonal average plot
+# Use geocat.viz.util convenience function to set axes limits & tick values
+gvutil.set_axes_limits_and_ticks(ax2,
+                                 xlim=[-0.6, 0.9],
+                                 ylim=[0, 1.55 * 1e16],
+                                 xticks=np.arange(-0.3, 0.7, 0.3),
+                                 yticks=np.linspace(0, 1.55 * 1e16, 7),
+                                 xticklabels=['-0.30', '', '0.30', ''],
+                                 yticklabels=np.arange(0, 181, 30))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax2,
+                             x_minor_per_major=1,
+                             y_minor_per_major=3,
+                             labelsize=12)
+
+# Remove tick marks on left side of ax2
+ax2.tick_params('y', which='both', left=False)
+
+# Use geocat.viz.util convenience function to add titles
+gvutil.set_titles_and_labels(ax2,
+                             maintitle="Zonal Ave",
+                             maintitlefontsize=12)
+
+# Plot zonal average
+ax2.plot(mean, times, linewidth=0.5, color='black')
+
+plt.show()
+
+##############################################################################
+# Define helper function to create the four subplots
+
+
+def make_subplot(fig, gridspec, xlim):
+    # Create axes for the contour plot and the zonal average plot
+    ax1 = fig.add_subplot(gridspec[0])
+    ax2 = fig.add_subplot(gridspec[1])
+
+    # Draw contour lines
+    ax1.contour(lon,
+                times,
+                chi,
+                levels=np.arange(-12, 13, 2),
+                colors='black',
+                linestyles='solid',
+                linewidths=.5)
+
+    # Draw filled contours, save the mappable to create colorbar later
+    cf = ax1.contourf(lon,
```

Comment:
```suggestion
    ax1.contourf(lon,
```
cf is defined and unused, but contourf is used. So just removing the variable definition works here

Reply:
`cf` is used to make the colorbar on line 250. I will change this function to return the color mappable so it can be used later